.github/workflows/inspektor-gadget.yml

```diff
@@ -144,7 +138,7 @@ jobs:
 
         kubectl_gadget=kubectl-gadget-${{ matrix.os }}-${{ matrix.arch }}
 
-        make $kubectl_gadget
+        CONTAINER_REPO="${{ env.REGISTRY }}/${{ env.CONTAINER_IMAGE }}" make $kubectl_gadget
```

Comment:
Is it possible to define this variable at the beginning to avoid repeating it multiple times? I think "Set IMAGE_TAG to environment" action could evolve in something more generic like "set build env variables" and put this logic there.

Reply:
I agree. Another option is to define `CONTAINER_REPO` instead of `CONTAINER_IMAGE` given that it is never used directly.

Reply:
This is what I tried to do.
But while defining the env you cannot use the env, _i.e._ this is not valid:

```yaml
env:
  foobar: barfoo
  quux: ${{ env.foobar }}/corge
```

But I will write it later like it is done for `IMAGE_TAG`.

> I think "Set IMAGE_TAG to environment" action could evolve in something more generic like "set build env variables" and put this logic there.

Why not, I will think about it because I would like to avoid generating conflicts overriding environment variables as much as possible.